crates/server/src/lib.rs

```diff
@@ -49,18 +41,18 @@ impl Config {
         self
     }
 
-    pub fn build_router(self) -> Result<Router> {
+    pub fn build_router(self) -> Router {
```

Comment:
This is fine for now, but I would bet this becomes fallible again as soon as `services::init` needs to connect to a database.

Reply:
Happy to restore it now; just didn't see any actual fallible calls taking place currently.

Reply:
No need, just a remark.